api/src/test/java/io/druid/timeline/DataSegmentTest.java

```diff
@@ -82,9 +82,15 @@ public ShardSpecLookup getLookup(List<ShardSpec> shardSpecs)
       }
 
       @Override
-      public Map<String, RangeSet<String>> getDomain()
+      public List<String> getDomainDimensions()
```

Comment:
Also optionally this can be removed and the full dimensionRangeCache from https://github.com/druid-io/druid/pull/5688/files#diff-63fc5dd623cea0f8e53e2e0961ff173eR128 can be passed in

Reply:
@niketh I don't quite follow you, what modification do you suggest here?